### ISSUE TITLE
transformation formulas give incorrect values for exponents, still

### ISSUE THREAD

**Issue by matthewParksViome** [tidyverse/ggplot2#3289]
Release 3.1.1 [claims to fix (#2978)](https://github.com/tidyverse/ggplot2/commit/47d65ddbb92066513578846512f98355d9b5e0d9) the tick misalignment issue for `sec_axis`, but it does not.

The minimal reproducible example from #3224 gives the same incorrect result with release 3.1.1.

**Comment by dpseidel** [comment]
This will be included in the forthcoming 3.2.0 release. Have you tried installing the current dev branch:


``` r
library(ggplot2)
sessioninfo::package_info(pkgs = "ggplot2", dependencies = FALSE)
#>  package * version    date       lib source
#>  ggplot2 * 3.1.1.9000 2019-05-02 [1] local 
#> 
#> [1] /Library/Frameworks/R.framework/Versions/3.5/Resources/library

df=data.frame(x=seq(from=0,to=-0.95,by=-0.05) , y=rnorm(20))

ggplot(data=df , aes( x = x/log(10) , y = y ) ) +
  geom_point() + 
  scale_x_continuous(sec.axis = sec_axis(~10^(.)) )
```

![](https://i.imgur.com/rB9nZKB.png)

<sup>Created on 2019-05-02 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by matthewParksViome** [comment]
the dev version is working great for me.
Thank you for your help!
